core/storage/user/gae_models.py

```diff
@@ -1904,6 +1904,12 @@ class UserQueryModel(base_models.BaseModel):
     edited_at_least_n_exps = datastore_services.IntegerProperty(default=None)
     # Query option to check if user has edited fewer than n explorations.
     edited_fewer_than_n_exps = datastore_services.IntegerProperty(default=None)
+    # Query option to check if user has created collection.
+    created_collection = datastore_services.BooleanProperty(default=False)
+    # Query option to check if user has used LogicProof interaction in any
+    # of the explorations.
```

Comment:
```suggestion
    # of the explorations that they created.
```
this might be better

Reply:
Done.